apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -884,6 +884,22 @@ func (m *ModelManagerAPI) ModelInfo(args params.Entities) (params.ModelInfoResul
 	return results, nil
 }
 
+// isModelAdmin reports if authorised user is either a controller admin
+// or has admin access to a given model.
+func (m *ModelManagerAPI) isModelAdmin(tag names.ModelTag) (bool, error) {
```

Comment:
Not your code, but I'd love a $ for each of these in the code base. I wish we had a re-usable set of helpers this type of thing.